src/static/js/webmentions.js

```diff
@@ -0,0 +1,331 @@
+// Code related to parsing and showing webmentions
+
+// Gets the webmentions json data from current URL
+async function getWebmentions(targetURL) {
+  const apiURL = `https://webmention.io/api/mentions.jf2?perPage=500&target=${targetURL}`;
+  let mentions = []
+  try {
+    const response = await window.fetch(apiURL);
+    if (response.status >= 200 && response.status < 300) {
+      const json = await response.json();
+      mentions = json.children;
+    } else {
+      console.error("Could not parse response", response.statusText);
+    }
+  } catch(error) {
+    console.error("Request failed", error);
+  }
+  return mentions;
+}
+
+// Format published date into human readable form
+function formatDate(dateString){
+  const options = { year: "numeric", month: "long", day: "numeric" }
+  return new Date(dateString).toLocaleDateString(undefined, options)
+}
+
+// Parse webmentions for individual category
+function parseMentions(webmentions, mentionType) {
+  let filteredMentions = []
+  webmentions.forEach(function(mention){
+    if (mention["wm-property"] == mentionType) {
+      filteredMentions.push(mention)
+    }
+  });
+  return filteredMentions;
+}
+
+// Renders webmention likes
+function renderLikes(likes) {
+  // Add the likes count to the likes-tab
+  document.querySelector('#likes-count').textContent = likes.length;
+  if (likes.length === 1) {
+    document.querySelectorAll('.like-singular').forEach(el => {
+      el.removeAttribute('data-translation');
+    });
+  } else {
+    document.querySelectorAll('.like-plural').forEach(el => {
+      el.removeAttribute('data-translation');
+    });
+  }
+
+  const likePanel = document.querySelector("#likes-panel");
+  const likeAriaText = likePanel.getAttribute("data-like-text");
+  const webmentionLikesList = document.createElement("ul");
+  webmentionLikesList.setAttribute("class", "webmention-likes");
+  likes.forEach(function(like) {
+    const likeHtml = `
+      <li class="webmention-likes-item">
+        <a
+          class="webmention-author"
+          href="${like["url"]}"
+          title="${like["author"]["name"]}"
+          aria-label="${like["author"]["name"]} ${likeAriaText}"
```

Comment:
> Shouldn't title and Aria Label be equal?

Not necessarily. Many assistive tools will announce both actually. aria-label in anchor tags are helpful to convey the actual purpose of the link when the anchor text doesn't. Hence my effort to add the text along with name. But I can see how it would complicate the translation. Still think maybe we should have an open ticket for such cases to brainstorm later?